.travis.yml

```diff
@@ -39,11 +39,6 @@ install:
 script:
   - gulp build
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then gulp cover:enableconfig; fi
-  - if [ $TRAVIS_OS_NAME == "osx" ]; then
```

Comment:
At some point we should readd the karma tests, they test the front end code.

Reply:
From the tests it looks like we just checked whether it loaded on chrome. Now that we use webviews, do we still need those tests?